get_streamlit_version.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+"""Extract Streamlit version from setup.py for use in build process."""
+
+import os
+import re
+import sys
+from pathlib import Path
+
+def get_streamlit_version():
+    """Extract VERSION string from setup.py."""
+    setup_py = Path(__file__).parent / "streamlit" / "lib" / "setup.py"
+    
+    if not setup_py.exists():
+        print(f"Error: Could not find setup.py at {setup_py}", file=sys.stderr)
+        sys.exit(1)
+    
+    try:
+        content = setup_py.read_text()
+        # Look for VERSION = "x.y.z" pattern
+        match = re.search(r'VERSION\s*=\s*["\']([^"\']+)["\']', content)
```

Comment:
Do you think the `ast`-based approach is more stable than this regex-based one? If so, improve the code.